deployment/media-insights-stack.yaml

```diff
@@ -1633,6 +1637,7 @@ Resources:
         MediaInsightsEnginePython36Layer: !Ref MediaInsightsEnginePython36Layer
         StartWaitOperationLambda: !GetAtt StartWaitOperationLambda.Arn
         CheckWaitOperationLambda: !GetAtt CheckWaitOperationLambda.Arn
+        Boto3UserAgent: !Join ['', ['{"user_agent_extra": "AwsSolution/', !Ref "SolutionId", '/%%VERSION%%"}']]
```

Comment:
I like this, it provides a good place to manually verify that the solution tag is correct, since you can see it on the inputs of the deployed stack.